extensions/oidc/runtime/src/test/java/io/quarkus/oidc/runtime/OidcUtilsTest.java

```diff
@@ -241,7 +241,8 @@ public void testTokenWithCustomRolesWrongPath() throws Exception {
     public void testTokenIsOpaque() throws Exception {
         assertTrue(OidcUtils.isOpaqueToken("123"));
         assertTrue(OidcUtils.isOpaqueToken("1.23"));
-        assertFalse(OidcUtils.isOpaqueToken("1.2.3"));
```

Comment:
This old test code shows that it was not even imaginable that a binary access token could have 2 dots, I have never seen it before, but that was wrong. But now it is checked properly